lib/autoyast.pm

```diff
@@ -63,10 +63,11 @@ sub expand_patterns {
         }
         elsif (is_sle('12+') && check_var('SLE_PRODUCT', 'sles')) {
             my @sle12;
-            push @sle12, qw(Minimal apparmor base documentation 32bit)                 if check_var('DESKTOP', 'textmode');
-            push @sle12, qw(Minimal apparmor base x11 documentation gnome-basic 32bit) if check_var('DESKTOP', 'gnome');
-            push @sle12, qw(desktop-base desktop-gnome)                                if get_var('SCC_ADDONS') =~ m/we/;
-            push @sle12, qw(yast2)                                                     if is_sle('>=12-sp3');
+            push @sle12, qw(Minimal apparmor base documentation)                 if check_var('DESKTOP', 'textmode');
+            push @sle12, qw(Minimal apparmor base x11 documentation gnome-basic) if check_var('DESKTOP', 'gnome');
+            push @sle12, qw(desktop-base desktop-gnome)                          if get_var('SCC_ADDONS') =~ m/we/;
+            push @sle12, qw(yast2)                                               if is_sle('>=12-sp3');
+            push @sle12, qw(32bit) unless check_var('ARCH', 'aarch64');
```

Comment:
I am not sure, how many `DESKTOP` variants we have, but this will add `32bit` without the `DESKTOP` variable (in contrast to previous behavior).

Reply:
True, added the DESKTOP we have defined here